assets/js/base/components/quantity-selector/style.scss

```diff
@@ -16,6 +19,13 @@
 	border: 1px solid $core-grey-light-600;
 	background: #fff;
 	border-radius: 4px;
+	// needed so that buttons fill the container.
+	box-sizing: content-box;
```

Comment:
This fixes this issue, it wasn't visible because when using 2px as the outline the width, the space is filled, but when using 1px for dark mode, it becomes visible, the picture below is when using 1px 
![image](https://user-images.githubusercontent.com/6165348/90143158-2df27d00-dd75-11ea-8798-cf80c5c85a15.png)